server/utils/utils.ts

```diff
@@ -94,3 +105,16 @@ export const dateAndTimeInputsAreValidDates = <K extends string | number>(
 
   return true
 }
+/**
+ * Retrieves response for a given question from the session object.
+ * @param sessionData the session data for an application.
+ * @param question the question that we need the response for in camelCase.
+ * @returns name converted to proper case.
+ */
+export const retrieveQuestionResponseFromSession = <T>(sessionData: Record<string, unknown>, question: string) => {
+  try {
+    return sessionData['basic-information'][kebabCase(question)][question] as T
+  } catch (e) {
+    throw new SessionDataError()
```

Comment:
Very much a fan of throwing specific errors here 👍 Would it be worth adding a message here? Something like `Question ${question} was not found in the session`.